scripts/add_directory.js

```diff
@@ -187,8 +188,14 @@ function submitBatch(n) {
                 break;
         }
         $("#alert-wrapper-" + n).html($(resAlert));
+        // activate tooltips
+        $("#alert-wrapper-" + n + ' [data-toggle="tooltip"]').tooltip();
         $(".glyphicon", buttons).toggleClass("hidden");
-    }, 1000)
+    });
+}
+
+function makeImgErrorHover(img, err) {
+    return `<a href="#" data-toggle="tooltip" title="${err}">${img}</a>`;

```

Comment:
It looks like you're making a link to an image here that goes nowhere? I think I'm confused what this does, what's the point of making this a link?